packages/flutter/lib/src/material/checkbox_list_tile.dart

```diff
@@ -57,6 +57,96 @@ import 'theme_data.dart';
 /// ```
 /// {@end-tool}
 ///
+/// ## CheckboxListTile isn't exactly what I want
+///
+/// If the way CheckboxListTile pads and positions its elements isn't quite
+/// what you're looking for, you can create custom labeled checkbox widgets by
+/// combining [Checkbox] with other widgets, such as [RichText] and [Padding].
+///
+/// {@tool snippet --template=stateful_widget_scaffold}
+///
+/// Here is an example of a custom LinkedLabelCheckbox widget that allows a
+/// [RichText] widget to be nested in a checkbox tile widget.
+///
+/// Since CheckboxListTile is wrapped with a [MergeSemantics] widget, it wants
```

Comment:
I updated it to be the following:
```
/// Since CheckboxListTile wraps its children with a [MergeSemantics] widget, it
/// wants to merge its descendant [Semantics] nodes to appear as just one
/// interactive node in the semantics tree.
///
/// Thus, CheckboxListTile will not work with children widgets that need to be \
/// a standalone interactive element. One example would be the [RichText]
/// widget, which has an embedded gesture recognizer configured to have
/// its own [Semantics] node. Thus, you may need to create a custom widget to
/// accommodate your needs.
```